plugins/opusCodec/src/OpusDecoder.h

```diff
@@ -0,0 +1,29 @@
+#ifndef OPUSDECODER_H
+#define OPUSDECODER_H
+
+
+#include <plugins/CodecPlugin.h>
+#include "opus/opus.h"
+
+
+class AthenaOpusDecoder : public Decoder {
+public:
+    AthenaOpusDecoder(int sampleRate, int numChannels);
+    ~AthenaOpusDecoder() override;
+
+
+    virtual void decode(const QByteArray& encodedBuffer, QByteArray& decodedBuffer) override;
+    virtual void lostFrame( QByteArray &decodedBuffer) override;
+
+
+private:
+    int _encodedSize;
+
+    OpusDecoder *_decoder = nullptr;
+    int _opus_sample_rate = 0;
```

Comment:
Please write variables in mixed case   like `_opus_sample_rate` should be `_opusSampleRate` Coding standard 1.1.2 https://github.com/kasenvr/project-athena/blob/kasen/core/CODING_STANDARD.md#112-private-class-variables-must-be-in-mixed-case-prefixed-with-an-underscore